src/v/kafka/server/tests/fetch_session_test.cc

```diff
@@ -175,8 +176,14 @@ FIXTURE_TEST(test_session_operations, fixture) {
     BOOST_TEST_MESSAGE("test updating session");
     {
         // Remove and forget about the first partition.
-        req.data.topics[0].fetch_partitions.erase(
+        auto parts = req.data.topics[0].fetch_partitions.copy();
+        parts.erase_to_end(
           std::next(req.data.topics[0].fetch_partitions.begin()));
+        std::move(
+          std::next(req.data.topics[0].fetch_partitions.begin(), 2),
+          req.data.topics[0].fetch_partitions.end(),
+          std::back_inserter(parts));
+        req.data.topics[0].fetch_partitions = std::move(parts);
```

Comment:
I don't think this does what you want it to.

Reply:
yikes